src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -11,5 +11,6 @@ public class CliSyntax {
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
+    public static final Prefix PREFIX_MODULE = new Prefix("r/");
```

Comment:
fyi i'll also be adding this prefix for my add command PR

i'll also use the same variable name then! my PR will have to be merged first so that you can test adding, so this will avoid merge conflict for this file i believe